src/com/oltpbenchmark/WorkloadConfiguration.java

```diff
@@ -47,20 +47,23 @@ public void setBenchmarkName(String benchmarkName) {
 	private String db_name;
 	private String db_username;
 	private String db_password;
-	private String db_driver;	
+	private String db_driver;
 	private double scaleFactor = 1.0;
 	private double selectivity = -1.0;
 	private int terminals;
 	private int loaderThreads = ThreadUtil.availableProcessors();
 	private int numTxnTypes;
     private TraceReader traceReader = null;
+    private boolean useKeyingTime = true;
```

Comment:
With these defaults, we'll use keying and think time for other workloads like `workload_1`, `workload_2` etc. also. Just want to make sure that it's intentional.

Reply:
workload_all should be what we should run now. And all of our defaults should be to make it run easy for that.
For the other workloads we can still make it false and then run it.